editor/assets/default_renderpipeline/builtin-pipeline.ts

```diff
@@ -137,1295 +123,1656 @@ function setupPipelineConfigs(
     configs.platform.w = (screenSpaceSignY * 0.5 + 0.5) << 1 | (device.capabilities.clipSpaceSignY * 0.5 + 0.5);
 }
 
+export interface PipelineSettings2 extends PipelineSettings {
+    _passes?: rendering.PipelinePassBuilder[];
```

Comment:
Will `_passes` be designed as a private property?
I found that all properties in PipelineSettings do not have a `_` prefix.